### DISCUSSION TITLE
Requesting an updated interval snapshot successful from one ring but not the other

### DISCUSSION THREAD

**Discussion by tismofied** [tsightler/ring-mqtt#465] [Support Q&A] [answered]
I have 2 ring doorbells. One in the back door and the other in the front.  I have everything setup per documentation.  Both have identical settings in the app. The backdoor ring is the only one requesting interval snapshot but not the front. 
Using HA as Vm on proxmox.

Home Assistant 2022.9.2
Supervisor 2022.08.6
Operating System 8.5
Frontend 20220907.0 - latest
Mosquitto broker
Current version: 6.1.3

Ps; for some reason also, the back camera interval reverts back to 30 even after I change it and the front ring reverts back to 600

**Comment by tsightler** [comment, marked as answer]
Based on the fact that one is defaulting to 30 seconds and the other is defaulting to 600 seconds, I'm assuming you must have two different models and one is a battery powered cameras.  As mentioned in the docs, the "auto" snapshot mode defaults to motion snapshots only for battery/low-power camera to avoid the additional battery drain.  If you want both motion and interval snapshots for a battery/low-power camera, you'll have to set the snapshot mode to "All" for that camera.

> Ps; for some reason also, the back camera interval reverts back to 30 even after I change it and the front ring reverts back to 600

This is pretty strange and I've had zero reports of this behavior, what do the addon logs show when you attempt to set the camera snapshot interval?

You didn't really share enough information to help you much more than that.  Ideally there would need to be full logs for at least 15 minutes, or at least some log snippets showing the behavior and specific details about your camera models.

**Reply by tismofied** [reply]
I am sorry I am not very technical, and I just started using GitHub. I'll try to get you more info and logs once I figure out how lol.
On the other hand, you did answer my question about why they are reverting to those intervals. I don't understand why though because I have them both wired to the house doorbell wiring with a new 16V, 30 VA transformer.  the front doorbell is a video doorbell 2 and it says hardwired in the app but still shows battery percentage. The backdoor ring is a regular video doorbell, and it says on the app "transformer voltage" 
I'll try the "ALL" option and see how it affects battery but then again, I cancelled the subscription (they jacked up prices) so maybe motion won't work for me?
please advise.

**Reply by tsightler** [reply]
The answers to these questions are covered in the hardware section of the [FAQ](https://github.com/tsightler/ring-mqtt/wiki/Frequently-Asked-Questions-(FAQ)#hardware).  The Video Doorbell 2 is still a battery powered device that only uses the available power to [trickle charge the battery](https://support.ring.com/hc/en-us/articles/204252980-Charging-your-Ring-Video-Doorbell-battery-with-hardwiring).

Without a Ring protect subscription, motion snapshots will not work with low-power cameras, but interval snapshots should be possible.

**Reply by tismofied** [reply]
Thank you for your time. I really appreciate it
